_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,199 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any single display orientation using CSS media queries.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS `transform` property with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) transform-function, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`landscape` or `portrait`).
```

Comment:
As a non-CSS-pro, there is something about the wording (and probably styling with/without backticks) of this, that doesn't really give me a clue about what kind of code we are looking for: 
"has a CSS `transform` property with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) transform-function, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`landscape` or `portrait`)."

I mean, I can get an idea from the examples, but it would be great also being able to make sense out of the applicability itself :)

Reply:
Ha ha, i re-read it many times now and it makes sense. We are simply suggesting look for a certain CSS with a specific property set.